astropy/io/fits/__init__.py

```diff
@@ -50,13 +50,19 @@ class Conf(_config.ConfigNamespace):
         'If True, use lazy loading of HDUs when opening FITS files by '
         'default; that is fits.open() will only seek for and read HDUs on '
         'demand rather than reading all HDUs at once.  See the documentation '
-        'for fits.open() for more datails.')
+        'for fits.open() for more details.')
     enable_uint = _config.ConfigItem(
         True,
         'If True, default to recognizing the convention for representing '
         'unsigned integers in FITS--if an array has BITPIX > 0, BSCALE = 1, '
         'and BZERO = 2**BITPIX, represent the data as unsigned integers '
         'per this convention.')
+    use_fsspec = _config.ConfigItem(
```

Comment:
Maybe make the default `None` so that `False` can mean to not use it at all, and `True` whenever possible. I think that is easier to understand.